python/lsst/daf/butler/core/formatter.py

```diff
@@ -72,10 +72,11 @@ class Formatter(metaclass=ABCMeta):
     are supported. These param (`frozenset`).
     """
 
+    # Should we use ... to say that the parameters are unconstrained?
```

Comment:
FWIW, mypy hates `...` and I had to work pretty hard to make it work with the usage of `...` in Registry queries; I wouldn't use it unless it's very nice as a user-facing thing (GvR recommends enums as sentinals instead).

Reply:
I did see that recent sphinx fix that showed me that type and that is what gave me pause.